ui/pages/confirmations/components/confirm/info/token-transfer/transaction-flow-section.tsx

```diff
@@ -0,0 +1,55 @@
+import { TransactionMeta } from '@metamask/transaction-controller';
+import React from 'react';
+import { ConfirmInfoSection } from '../../../../../../components/app/confirm/info/row/section';
+import {
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+} from '../../../../../../components/component-library';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  IconColor,
+  JustifyContent,
+} from '../../../../../../helpers/constants/design-system';
+import { useConfirmContext } from '../../../../context/confirm';
+import { useDecodedTransactionData } from '../hooks/useDecodedTransactionData';
+import { ConfirmLoader } from '../shared/confirm-loader/confirm-loader';
+import { NameOrAddressDisplay } from './name-or-address-display';
+
+export const TransactionFlowSection = () => {
+  const { currentConfirmation: transactionMeta } =
+    useConfirmContext<TransactionMeta>();
+
+  const { value, pending } = useDecodedTransactionData();
+
+  const recipientAddress =
+    value?.data[0].params.find((param) => param.type === 'address')?.value ||
+    '0x0000000000000000000000000000000000000000';
```

Comment:
Is this a safe fallback or should we let an error happen?

Reply:
That's a good point, I removed the fallback.